README.md

```diff
@@ -1,2 +1,24 @@
 # e2e-nativek8s
-OpenEBS E2E Pipelines
+
+This repository's `release-branch` branch contains plateform execution scripts for each indivisual test which is meant to run for inducing chaos and testing zfs-localpv components. It includes several stages to perform different scenraios in end to end work-flow. To see the README's and test results of pipeline switch to the `release-branch` of this repository.
```

Comment:
typos 
** indivisual --> individual
** scenraios --> scenario

for readme status and test results need to move master or release branch?